nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/ListS3.java

```diff
@@ -181,7 +181,7 @@ public class ListS3 extends AbstractS3Processor implements VerifiableProcessor {
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .required(false)
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .description("The prefix used to filter the object list. In most cases, it should end with a forward slash ('/').")
+            .description("The prefix used to filter the object list. Do not begin with a forward slash ('/').")
```

Comment:
Instead of rewording the description, it seems better to add the additional sentence, and the parentheses can be removed.
```suggestion
            .description("The prefix used to filter the object list. Do not begin with a forward slash '/'. In most cases, it should end with a forward slash '/'.")
```